autoload/intero/process.vim

```diff
@@ -67,7 +72,9 @@ function! intero#process#initialize() abort
             call s:start_compile(10, l:opts)
         else
             let g:intero_built = 1
-            call intero#process#start()
+            if g:intero_start_immediately
```

Comment:
Just made one more change. When we move `intero_start_immediately` to guard against the `stack build` call, we need to make sure that there's a way for the user to trigger the `stack build` when they're ready.

I implemented this by checking that intero is initialized every time we call `intero#process#start()`. That way, the user can just run `:InteroStart` and if we need to build the backend, it'll get built automatically.

*Now* the PR should be ready.